packages/pwa-kit-react-sdk/scripts/file-utils.test.js

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Would be preferable to disable this by directory, (e.g. `*/scripts/*`) and filename convention `eslint.config.js` I thought that was possible with eslint?